build/ical.js

```diff
@@ -808,6 +811,7 @@ ICAL.design = (function() {
       },
 
       "action": DEFAULT_TYPE_TEXT,
+      "allday": DEFAULT_TYPE_BOOLEAN,
```

Comment:
allday is not expressed through an extra property, but instead by using the date fields with VALUE=DATE.